extensions/compose/src/cli-run.ts

```diff
@@ -146,10 +148,16 @@ export class CliRun {
       destinationPath = path.join(os.homedir(), 'AppData', 'Local', 'Microsoft', 'WindowsApps', `${binaryName}.exe`);
       command = ['copy', `"${binaryPath}"`, `"${destinationPath}"`];
     } else {
-      destinationPath = path.join('/usr/local/bin', binaryName);
+      destinationPath = path.join(localBinDir, binaryName);
       command = ['cp', binaryPath, destinationPath];
     }
 
+    // If on macOS or Linux, check to see if the /usr/local/bin directory exists,
+    // if it does not, then add mkdir -p /usr/local/bin to the start of the command when moving the binary.
+    if (system === 'linux' || (system === 'darwin' && !fs.existsSync(localBinDir))) {
```

Comment:
it works on linux because of the `-p` flag but i wonder why you didn't write as 
```
(system === 'linux' || system === 'darwin') && !fs.existsSync(localBinDir)
```

Reply:
Ahhh good catch. That's totally a mistake there.